<ui version="4.0" >
 <author>Lukas Tinkl &lt;lukas@kde.org></author>
 <class>KThemeDlg</class>
 <widget class="QWidget" name="KThemeDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>409</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" >
   <property name="sizeConstraint" >
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text" >
        <string>Choose your visual KDE theme:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>170</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KUrlLabel" name="lbGet" >
       <property name="text" >
        <string>Get new themes...</string>
       </property>
       <property name="url" >
        <string comment="KDE::DoNotExtract">http://themes.kde.org</string>
       </property>
       <property name="tipText" >
        <string>Go to the KDE themes website</string>
       </property>
       <property name="useTips" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item rowspan="4" row="0" column="1" >
      <widget class="QLabel" name="lbPreview" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QPushButton" name="btnRemove" >
       <property name="text" >
        <string>&amp;Remove Theme</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QPushButton" name="btnCreate" >
       <property name="text" >
        <string>Create &amp;New Theme...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QPushButton" name="btnInstall" >
       <property name="text" >
        <string>&amp;Install New Theme...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="Q3ListView" name="lvThemes" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="showSortIndicator" >
        <bool>true</bool>
       </property>
       <property name="resizeMode" >
        <enum>Q3ListView::AllColumns</enum>
       </property>
       <column>
        <property name="text" >
         <string>Theme</string>
        </property>
        <property name="clickable" >
         <bool>true</bool>
        </property>
        <property name="resizable" >
         <bool>false</bool>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="Line" name="line1" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="textLabel1_2" >
     <property name="text" >
      <string>Customize your theme:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QToolButton" name="btnBackground" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>60</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Customize the desktop background</string>
       </property>
       <property name="text" >
        <string>Background</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnColors" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>60</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Customize colors</string>
       </property>
       <property name="text" >
        <string>Colors</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnStyle" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>60</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Customize the widget style</string>
       </property>
       <property name="text" >
        <string>Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnIcons" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>60</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Customize the icon theme</string>
       </property>
       <property name="text" >
        <string>Icons</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnFonts" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>60</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Customize the font theme</string>
       </property>
       <property name="text" >
        <string>Fonts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSaver" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>60</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Customize the screen saver</string>
       </property>
       <property name="text" >
        <string>Screen Saver</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lvThemes</tabstop>
  <tabstop>btnInstall</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnCreate</tabstop>
  <tabstop>btnBackground</tabstop>
  <tabstop>btnColors</tabstop>
  <tabstop>btnStyle</tabstop>
  <tabstop>btnIcons</tabstop>
  <tabstop>btnFonts</tabstop>
  <tabstop>btnSaver</tabstop>
 </tabstops>
 <includes>
  <include location="local" >kurllabel.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
